tests/test_blurb_add.py

```diff
@@ -0,0 +1,218 @@
+import re
+
+import pytest
+
+from blurb import blurb
+
+
+ALLOWED_SECTION_IDS = list(map(str, range(1 + len(blurb.sections), 1)))
```

Comment:
Another reason for explicitly listing things in tests: this is an empty list, which causes the test below to be skipped.

```
tests/test_blurb_add.py::test_valid_section_id[section0] SKIPPED (got empty parameter set ['section'], function test_valid_section_id at /Users/hugo/github/blurb/tests/test_blurb_add.py:86)                      [ 48%]
```

Reply:
Ah ! indeed. :D You convinced me then !

Reply:
Actually, now I understand why the coverage told me that the lines were not covered!